npm/config/config.go

```diff
@@ -51,6 +51,7 @@ var DefaultConfig = Config{
 		ApplyInBackground: true,
 		// NetPolInBackground is currently used in Linux to apply NetPol controller Add events in the background
 		NetPolInBackground: true,
+		EnableNPMLite:      true,
```

Comment:
This default config should always be false, or else we will break existing deployments where the configmap is not explicitly set to false.

Reply:
Yup I had just enabled it for testing purposes with the intent of disabling it before merging the pr in. I can disable it in the pr and enable locally for testing purposes if that is the better way @vakalapa